### DISCUSSION TITLE
Hello, is there anybody out there?

### DISCUSSION THREAD

**Discussion by gildesmarais** [html2rss/html2rss.github.io#400] [General] [answered]
Out of curiosity, I wonder… who of you uses something that html2rss provides?

Let's get the discussions started! 

Is there anything (you think) which could be improved?
What would be something you'd like to contribute, but had troubles with?
What could html2rss do to improve the UX users have when using RSS?

Dear internet people: I look forward to hearing from you! :-)

**Comment by jvmiles** [comment]
It is such a shame that feedity is not available anymore. I am looking for a (self-hosted) replacement. 

1. OK, can I get this to run without docker? 

2. Is there a place where we can discuss configs for Websites?

**Reply by gildesmarais** [reply]
Hey, not sure what happened to feedity. 🤷🏽 

1. Sure, i.e. described in https://github.com/html2rss/html2rss#installation / requires Ruby to be installed in [your preferred way](https://www.ruby-lang.org/en/documentation/installation/). 
2. That place for now would be here. Just open a topic and wait for it to get traction.

Cheers

**Comment by mizzunet** [comment]
@gildesmarais Wish the instance could load config from external repos. 

Say, I've forked the config repo (github.com/mizzunet/html2rss-config) and made some of configs on my own there. Then, let public instance access them. Like https://1.h2r.workers.dev/external/github.com/mizzunet/html2rss-config/cnn/new_stories.rss

Thanks

**Reply by gildesmarais** [reply]
Supporting more config repos sounds sensible to. I'm also not quite happy with the cumbersome approach `html2rss-configs` takes. I'd prefer to just create a yml and would directly profit from automatic testing with rspec. Also taking the yml-files huckepack in a gem doesn't feel that *correct*.

So, I'm open for other 'registry' approaches. Let's think a bit architecture first before one writes code for it.

However, to manage expectations: "the" public instance will certainly never accept just any "random" config repository. But that should not matter, as everybody can host their own instance and - when they're nice - share it with everyone. :)

**Reply by gildesmarais** [reply]
Added this request to https://github.com/orgs/html2rss/projects/3/views/1

**Comment by gildesmarais** [comment, marked as answer]
Thanks for saying Hello. :-)

I've locked the topic (reactions are still allowed 👋🏽 ) to prevent a "mega thread" which discusses multiple ideas. Please [create a new Discussion](https://github.com/orgs/html2rss/discussions/new/choose) for these ideas and keep the board structured.
